src/app/countriesCard/Card.tsx

```diff
@@ -0,0 +1,36 @@
+import Image from "next/image"
+
+interface Propstype  {
+  flag : string,
+  alt : string,
+  name : string,
+  population : string,
+  region : string,
+  capital : string,
+}
+export default function CountriesCard (props : Propstype)  {
+
+  return (
+    <>
+      <div className=" w-80  h-custom-height shadow-lg rounded-xl bg-white cursor-pointer" key={props.cca3}>
```

Comment:
Avoid improper gaps like these ones in your classnames. @jakemelvin let's set up a linter in this with format on save